packages/backend/server/schema.prisma

```diff
@@ -443,6 +447,25 @@ model AiPrompt {
   @@map("ai_prompts")
 }
 
+model AiSession {
+  id          String   @id @default(uuid()) @db.VarChar
+  userId      String   @map("user_id") @db.VarChar
+  workspaceId String   @map("workspace_id") @db.VarChar
+  docId       String   @map("doc_id") @db.VarChar
+  promptName  String   @map("prompt_name") @db.VarChar
+  action      Boolean  @db.Boolean
```

Comment:
what is a boolean action?

Reply:
`action` is a bussiness definition, that means the chat is oneshot chat like translate

Reply:
better give it a  comment